README.md

```diff
@@ -224,9 +224,9 @@ This is how fields in the Identify Spec are populated:
 
 | Identify Spec field | Source field in Fingerprint Pro `result` or metadata passed to `segment.identify`                                  |
 | ------------------- | ------------------------------------------------------------------------------------------------------------------ |
-| `anonymousId`       | [`result.visitorId`](https://dev.fingerprint.com/docs/js-agent#visitorid) - Fingerprint `visitorId`                |
-| `traits.visitorId`  | [`result.visitorId`](https://dev.fingerprint.com/docs/js-agent#visitorid) - Fingerprint `visitorId`                |
-| `traits.createdAt`  | [`result.firstSeenAt.subscription`](https://dev.fingerprint.com/docs/js-agent#firstseenat) - time of first visit   |
+| `anonymousId`       | [`result.visitorId`](https://dev.fingerprint.com/reference/get-function#visitorid) - Fingerprint `visitorId`                |
+| `traits.visitorId`  | [`result.visitorId`](https://dev.fingerprint.com/reference/get-function#visitorid) - Fingerprint `visitorId`                |
+| `traits.createdAt`  | [`result.firstSeenAt.subscription`](https://dev.fingerprint.com/reference/get-function#firstseenat) - time of first visit   |
```

Comment:
Is the text inside documentation for firstSeenAt updated after our DRN product? I believe DRN changed how firstSeenAt's value